metaStoreApp/db/migrate/20200229142801_add_default_duration.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultDuration < ActiveRecord::Migration[5.2]
+  def change
+    execute "alter table media alter duration set default 0;"
```

Comment:
Do not write direct sql queries unless its absolutely required. Please try to use activerecord methods to set default value.

When you write a migration, you should have a clear idea of how to revert that particular migration. if we have to rollback this change, how will we do that?
Please read about the difference between `change`, `up` and `down` methods in rails migrations. 

Incase if you are writing a SQL query, please format and escape the query properly:

```
ALTER TABLE `media` ALTER `duration` SET DEFAULT 0;
```

lets say in future mysql adds media or duration as SQL keywords(similar to `SELECT` or `WHERE`) your code will break if its not escaped properly.

Reply:
okay